src/index-metadbs/mp/links.json

```diff
@@ -0,0 +1,34 @@
+{
+    "meta": {
+        "api_version": "1.0.0",
+        "query": {
+            "representation": "/links"
+        },
+        "more_data_available": false,
+        "schema": "https://schemas.optimade.org/openapi/v1.0/optimade_index.json"
+    },
+    "data": [
+        {
+            "id": "mp",
+            "type": "links",
+            "attributes": {
+                "name": "The Materials Project",
+                "description": "The Materials Project OPTIMADE endpoint",
```

Comment:
Just a comment, but you're welcome to do a more fulfilling description here, i.e., what databases/data is expected to be retrieved from Materials Project, or just a generic description of Materials Project :)